config/vufind/Blender.ini

```diff
@@ -188,8 +191,8 @@ lightboxLimit = 100 ; page size for the lightbox
 top_rows = 2
 ; Should we show "exclude" links for some or all of the facets? Set to * for
 ; all facets, use a comma-separated list to show for some of the facets, set
-; to false or omit to disable "exclude" links
-;exclude = *
+; to false or omit to disable "exclude" links.
+exclude = blender_backend
```

Comment:
Did you mean to commit the change to this line? Seems like it doesn't make sense as a default setting, since the field is disabled by default above.

Reply:
I think it still makes sense to allow exclude by default if the field is enabled. I can revert if you feel strongly about this, but it's easy to miss the options if they're not set by default, and I think the option is really handy for this facet.

Reply:
That's fair, I just wanted to be sure this was intentional. I almost wonder if we should turn exclude * on by default everywhere, since it's very useful in general... but that's a whole other conversation.